### PR TITLE
Add redux

### PR DESCRIPTION
**Pull request by Smarticles101**

### CODE REVIEW

File: src/configs/api.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+const currentDate = new Date();
+
+export const pullVitalData = toastCB => {
+  const { access_token } = store.getState();
+  const { user } = store.getState();
+
+  const uid = user.uid;
+  const docRef = db.collection("users").doc(uid);
+
+  if (access_token) {
```
Comment (KegenGuyll) [inline src/configs/api.js]: This won't ever run because you don't have access_token

**Comment by Sfredr01** [comment]
^^ that was because you made changes while i was mid push

File: src/configs/history.js [hunk 2]
```diff
@@ -0,0 +1,2 @@
+import { createBrowserHistory } from "history";
```
Comment (KegenGuyll) [inline src/configs/history.js]: What is this being used for?

Comment (Smarticles101) [reply src/configs/history.js]: Custom history used in the `api.js` file

**Pull request merged**